build(deps): bump loader-utils from 2.0.2 to 2.0.4 (#828)
redpanda-data/documentation: modules/get-started/pages/whats-new.adoc

* xref:redpanda-cloud:get-started:whats-new-cloud.adoc[] 
* xref:redpanda-cloud:get-started:cloud-overview.adoc#redpanda-cloud-vs-self-managed-feature-compatibility[Redpanda Cloud vs Self-Managed feature compatibility]

== Declarative user and ACL management in Kubernetes

> Redpanda Operator version v2.2.2-24.2.4 onwards supports the declarative management of users and access control lists (ACLs) using the new User resource. This feature allows you to:

Reviewer: ```suggestion
Starting with Redpanda Operator version v2.2.2-24.2.4, you can now manage users and access control lists (ACLs) with the new User resource. This feature allows you to:
```
